### ISSUE TITLE
Make up/down buttons in category sorting menu long-holdable

### ISSUE THREAD

**Issue by Xymanek** [X2CommunityCore/xcom2-launcher#68]
Moving categories around is a bit of a hurdle right now

**Comment by RevZero** [comment]
I do not think that this would improve usability, unless you have a really large amount of categories. The problem with long-holdable buttons is the tradeoff between 
- keeping the delay short enough to make it useful (clicking 10 times manually is probably better than waiting 10 x 1s = 10s)
- keeping the delay long enough to prevent missing the correct position (and having to correct it)

So my vote is on "wontfix" ;)

**Comment by Xymanek** [comment]
I have about 15-20 and reorganizing them was a pain once I found out that it was possible.

However, your points are valid. Perhaps a better solution is to make them draggable?

**Comment by RevZero** [comment]
Implementing drag/drop with the .NET default ListView control is really pain and not trivial to implement. 

If we REALLY need this, the category ListView should be replaced by an ObjectListView (as used by the mod list). But this would obviously require some additional work as well.

**Comment by RevZero** [comment]
@Xymanek : I will close this for now if that is OK with you and if no one else absolutely needs this. After all, sorting categories is not something you do every day.
